incubator/cassandra/templates/cassandra-statefulset.yaml

```diff
@@ -69,7 +69,7 @@ spec:
         {{- $seed_size := default 1 .Values.config.seed_size | int -}}
         {{- $global := . }}
         - name: CASSANDRA_SEEDS
-          value: "{{- range $i, $e := until $seed_size }}{{ template "cassandra.fullname" $global }}-{{ $i }}.{{ template "cassandra.fullname" $global }}.{{ $global.Release.Namespace }}.svc.cluster.local,{{- end }}"
+          value: "{{- range $i, $e := until $seed_size }}{{ template "cassandra.fullname" $global }}-{{ $i }}.{{ template "cassandra.fullname" $global }}.{{ $global.Release.Namespace }}.{{ $global.Values.clusterDomainSuffix }},{{- end }}"
```

Comment:
That's an unrelated change that should not be part of this PR.

Reply:
you're blazing fast. are you a robot? just fixed this :)